### PR TITLE
Kosaraju's algorithm c++

### PR DESCRIPTION
**Pull request by devanshiv123**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added Kosaraju's Algorithm in C++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/graphs/Kosaraju_Algorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#include<bits/stdc++.h>
```
Comment (iamrajiv) [inline C-Plus-Plus/graphs/Kosaraju_Algorithm.cpp:3]: @devanshiv123 add a comment `Implementation of Kosaraju's Algorithm to find out the strongly connected
   components (SCCs) in a graph.`

File: C-Plus-Plus/graphs/README.md [hunk 2]
```diff
@@ -1 +1,2 @@
 ### Graph algorithms in C++
+- [Kosaraju's Algorithm](Kosaraju_Algorithm.cpp)
```
Comment (iamrajiv) [inline C-Plus-Plus/graphs/README.md]: @devanshiv123 do not update `graphs/README.md` file

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@devanshiv123 add sample input-output. Also, add space and time complexity.

File: C-Plus-Plus/graphs/README.md (hunk 2 above)
Comment (devanshiv123) [reply C-Plus-Plus/graphs/README.md]: @iamrajiv Did the changes. Please review again.

**Review by atarax665 — APPROVED** [review]
LGTM :)

**Review by iamrajiv — APPROVED** [review]
@devanshiv123 LGTM!

**Review by siddharth25pandey — APPROVED** [review]
@devanshiv123 Good to go only resolve the merge conflicts error
Than it is able to merge
Thanks!

**Comment by devanshiv123** [comment]
@siddharth25pandey  I didnt get how to resolve merge conflicts. Can you please provide a tutorial or something related?

**Comment by siddharth25pandey** [comment]
> @siddharth25pandey I didnt get how to resolve merge conflicts. Can you please provide a tutorial or something related?

@devanshiv123 I have mentioned you in a thread on Slack Channel, do visit.

**Comment by devanshiv123** [comment]
@siddharth25pandey  I did try those commands but I am getting some errors in pushing the branch later on.

**Comment by devanshiv123** [comment]
I have made a new PR now. Please review it.

**Comment by iamrajiv** [comment]
@all-contributors please add @devanshiv123 for code

**Comment by siddharth25pandey** [comment]
@iamrajiv , As this PR has more number of commits than [this](https://github.com/TesseractCoding/NeoAlgo/pull/811) which is actually not required. Anyways you have merged it. NP

**Pull request merged**